src/components/config/OidcConfig.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { AuthProvider, User } from 'oidc-react';
```

Comment:
oidc-react package is missing in the package.json and package-lock.json

Reply:
fixed.